### DISCUSSION TITLE
Android: Can't choose PR reviewer

### DISCUSSION THREAD

**Discussion by htcfreek** [community/community#133354] [Mobile]
If I like to request a review for my PR using GH mobile app on Android I can't choose a reviewer.

1. I get the error message "You are not allowed to see other project contributiors." => Makes no sense as I can add various people using GH homepage.

2. On the GH homepage I can choose every people as reviewer. => Why does GH mobile app does not support this?

This makes it impossible to request reviews.

Additional information:
- Example repository: MS PowerToys
- Repository permission: Triage permission 
- App: GH mobile (latest stable release)
- OS: Android 11

**Comment by stevepopovich** [comment]
Hi @htcfreek we could definitely be handling this permission set wrong. Can I confirm the steps here:
1. Go to a PR you have `triage` permission for the repository for
2. Go to `Reviews` -> `REQUEST REVIEWS`
3. The screen shows `You are not allowed to see other project contributors.` so you can't pick reviewers.

Is that correct?

**Reply by htcfreek** [reply]
In general yes. 

But I made a mistake on the error message. The correct one is "You don't have the permission to see the repository collaborators.".

And I see the "Add reviewers" button only on my own PRs. PRs from other people don't show the button and requesting reviews fail in the first step pressing the button.

**Reply by stevepopovich** [reply]
Cool. Let me file a bug and pass it on to the team. Thanks!

**Reply by htcfreek** [reply]
I think these are two bugs:
1. I can't select reviewers.
2. I don't have a "request review" button on PRs I didn't create. (Only on my own PRs.)

_Both things work on the website for repositories where I have triage permissions._

**Reply by htcfreek** [reply]
Any updates on these two bugs?

**Reply by stevepopovich** [reply]
They are in our queue to be fixed. Thanks

**Reply by htcfreek** [reply]
Any updates on this? It's really annoying.

**Reply by stevepopovich** [reply]
At this time, we have not fixed this issue. Sorry, we will try to get to it in the future.

**Reply by htcfreek** [reply]
Any updates on this? It's really annoying.

**Reply by stevepopovich** [reply]
Sorry, still in our backlog.
